parsl/providers/kubernetes/kube.py

```diff
@@ -181,6 +180,93 @@ def _status(self):
         return jobs_ids
         # do something to get the deployment's status
 
+    def _create_pod(self,
+                    image,
+                    pod_name,
+                    job_name,
+                    port=80,
+                    cmd_string=None,
+                    volumes=[]):
+        """ Create a kubernetes non-deployment pod for the job.
```

Comment:
Refer to this as this a "pod", not a "non-deployment pod".

Reply:
fixed